x/datadeal/keeper/deal_test.go

```diff
@@ -0,0 +1,71 @@
+package keeper_test
```

Comment:
Add a test when dataSale's status is FAILED.  :)

Reply:
And it would be good to add a test when dataSale's status is `XXX_VOTING_PERIOD` and `COMPLETED`.
SellData should fail in this situation.